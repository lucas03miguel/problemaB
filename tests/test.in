3
7 7 
1 1
7 7 
2 2
7 7 
3 3
