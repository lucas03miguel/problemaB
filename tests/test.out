5040
3110940
68938800
